TASVideos.Test/ViewComponents/ParamHelperTests.cs

```diff
@@ -92,14 +109,12 @@ public void GetYear(string parameterStr, string param, int? expected)
 		[DataRow("csv=1,2,notnumber", "csv", new[] { 1, 2 })]
 		public void GetInts(string parameterStr, string param, int[] expected)
 		{
-			var actual = ParamHelper.GetInts(parameterStr, param);
+			var moduleString = "foo|" + parameterStr;
+			var module = new Module(0, moduleString.Length, moduleString);
+			var actual = WikiMarkup.ConvertParameter<IList<int>>(module.Parameters.GetValueOrDefault(param));
 
 			Assert.IsNotNull(actual);
-			var actualList = actual.ToList();
-			foreach (var expectedVal in expected)
-			{
-				Assert.IsTrue(actualList.Contains(expectedVal));
-			}
+			Assert.IsTrue(actual.SequenceEqual(expected));
```

Comment:
Sequence equal only returns true if it is the same values in the same order.  The previous code, while more verbose, ensures that they can be in a different order and still qualify as passing the requirements.

Reply:
Yes, but the callee gets passed an `IList<int>`, so I have to put things in _some_ order.  If we want these all to be order-indifferent at the API level we'll need to switch to `ISet<int>`.  (Or maybe, some are `ISet<int>` and some are `IList<int>` and we have two different parameter binders and two different sets of tests.)

If we really do want the callee to get an `IList<int>`, then I think `SequenceEqual` is appropriate and correct due to principle of least surprise:  If the incoming string is `1,3`, as a callee I'd expect `l[0] == 1, l[1] == 3`.

Reply:
Fair enough